code/__DEFINES/preferences.dm

```diff
@@ -22,22 +22,21 @@
 #define PREFTOGGLE_CHAT_DEBUGLOGS 			128
 #define PREFTOGGLE_CHAT_LOOC 				256
 #define PREFTOGGLE_CHAT_GHOSTRADIO 			512
-#define PREFTOGGLE_SHOW_TYPING 				1024
-#define PREFTOGGLE_DISABLE_SCOREBOARD 		2048
```

Comment:
Absolutely fucking not. This will corrupt all current bitflag entries.

Reply:
Does shifting them all down by one not stop that?
Surely unless something is checking for, say, `262144` rather than `PREFTOGGLE_DISABLE_KARMA` everything will work the same way since it's all defined.

Reply:
It shifts them down ingame, but not in the DB. 1024 in the DB before this will equal typing, but will equal scoreboard after, breaking everyones prefs.